pyproject.toml

```diff
@@ -130,6 +130,9 @@ ipython-genutils = { version = "^0.2.0", optional = true }
 requests-ftp = "^0.3.1"
 zarr = {version = "^2.7.0", optional = true, markers = "sys_platform != 'darwin' or (sys_platform == 'darwin' and platform_machine != 'arm64')"}  # not supported through numcodecs
 xarray = {version = "^0.17.0", optional = true}
+measurement = "^3.2.0"
+fuzzywuzzy = {version = "^0.18.0", extras = ["python-Levenshtein"]}
+python-Levenshtein = {version = "^0.12.2", optional = true}
```

Comment:
May I suggest to use the `levenshtein` package [1] instead? It has way better coverage of wheels for different targets, so installation will not require a compiler. Things like this cannot be expected to work on production machines.

[1] https://pypi.org/project/levenshtein/